crash/panic app_service_managed_certificate_resource

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

### Terraform (and AzureRM Provider) Version

Terraform v1.0.6
on windows_amd64
+ provider registry.terraform.io/cloudflare/cloudflare v2.26.1
+ provider registry.terraform.io/hashicorp/azuread v2.2.1
+ provider registry.terraform.io/hashicorp/azurerm v2.76.0
+ provider registry.terraform.io/hashicorp/random v3.1.0
+ provider registry.terraform.io/microsoft/azuredevops v0.1.7

### Affected Resource(s)

this appears to affect any code that has azurerm_app_service or linked resources eg app_service_managed_certificate_resource


### Panic Output

╷
│ Error: Plugin did not respond
│
│   with azurerm_app_service_certificate_binding.api,
│   on novf.tech.manged.cert.binding.tf line 78, in resource "azurerm_app_service_certificate_binding" "api":
│   78: resource "azurerm_app_service_certificate_binding" "api" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with azurerm_app_service_managed_certificate.ui,
│   on novf.tech.manged.cert.binding.tf line 171, in resource "azurerm_app_service_managed_certificate" "ui":
│  171: resource "azurerm_app_service_managed_certificate" "ui" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with azurerm_app_configuration_key.invz_redis_primary_access_key,
│   on settings.invz.tf line 211, in resource "azurerm_app_configuration_key" "invz_redis_primary_access_key":
│  211: resource "azurerm_app_configuration_key" "invz_redis_primary_access_key" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with azurerm_app_service_virtual_network_swift_connection.investorzone-app-api,
│   on vnet-intergration.tf line 1, in resource "azurerm_app_service_virtual_network_swift_connection" "investorzone-app-api":
│    1: resource "azurerm_app_service_virtual_network_swift_connection" "investorzone-app-api" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-azurerm_v2.76.0_x5.exe plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x0 pc=0x5ce4d4a]

goroutine 2041 [running]:
github.com/hashicorp/terraform-provider-azurerm/internal/services/web.resourceAppServiceManagedCertificateRead(0xc002887700, 0x61032c0, 0xc001332000, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/internal/services/web/app_service_managed_certificate_resource.go:221 +0x70a
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).read(0xc001305dc0, 0x710f5a8, 0xc0013b1180, 0xc002887700, 0x61032c0, 0xc001332000, 0x0, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:335 +0x1fb
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).RefreshWithoutUpgrade(0xc001305dc0, 0x710f5a8, 0xc0013b1180, 0xc000788bd0, 0x61032c0, 0xc001332000, 0xc001a155e8, 0x0, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/resource.go:624 +0x1cb
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ReadResource(0xc0013dc048, 0x710f5a8, 0xc0013b1180, 0xc0013b11c0, 0xc0013b1180, 0x9ebeca, 0x643b360)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema/grpc_provider.go:575 +0x45b
github.com/hashicorp/terraform-plugin-go/tfprotov5/server.(*server).ReadResource(0xc001bd1260, 0x710f650, 0xc0013b1180, 0xc0016964e0, 0xc001bd1260, 0xc001d2d7d0, 0xc002647ba0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/server/server.go:298 +0x108
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ReadResource_Handler(0x66045c0, 0xc001bd1260, 0x710f650, 0xc001d2d7d0, 0xc001696480, 0x0, 0x710f650, 0xc001d2d7d0, 0xc002081c00, 0x356)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:344 +0x222
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0002c8380, 0x715f298, 0xc0001d8900, 0xc00152f200, 0xc000c79440, 0xa82c4b0, 0x0, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:1292 +0x52b
google.golang.org/grpc.(*Server).handleStream(0xc0002c8380, 0x715f298, 0xc0001d8900, 0xc00152f200, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:1617 +0xd0c
google.golang.org/grpc.(*Server).serveStreams.func1.2(0xc0012f2c20, 0xc0002c8380, 0x715f298, 0xc0001d8900, 0xc00152f200)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:940 +0xb2
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/terraform-providers/terraform-provider-azurerm/vendor/google.golang.org/grpc/server.go:938 +0x1fd

Error: The terraform-provider-azurerm_v2.76.0_x5.exe plugin crashed!


### Important Factoids

we have tried it on a number of code sections in our repo all of which planned and applied ok 13th Sepetmber 2021
However as of this morning 14 September they all give the same panic.

No code has changed and we have not updated the azurerm provider
I tried downgrading azurerm provider to v2.25.0 and the same error occurs


### References



* #0000

--- a/internal/services/web/app_service_managed_certificate_resource.go
+++ b/internal/services/web/app_service_managed_certificate_resource.go
@@ -218,7 +218,11 @@ func resourceAppServiceManagedCertificateRead(d *pluginsdk.ResourceData, meta in
 		d.Set("subject_name", props.SubjectName)
 		d.Set("host_names", props.HostNames)
 		d.Set("issuer", props.Issuer)
-		d.Set("issue_date", props.IssueDate.Format(time.RFC3339))
+		issueDate := ""
+		if props.IssueDate != nil {
+			issueDate = props.IssueDate.Format(time.RFC3339)
+		}
+		d.Set("issue_date", issueDate)
 		expirationDate := ""
 		if props.ExpirationDate != nil {
 			expirationDate = props.ExpirationDate.Format(time.RFC3339)

